blender/arm/material/make_tess.py

```diff
@@ -24,6 +24,7 @@ def interpolate(tese, var, size, normalize=False, declare_out=False):
 
     if normalize:
         s += '{0}{1} = normalize({1}_0 + {1}_1 + {1}_2);\n'.format(prep, var)
+        s += 'vec3 n = normalize({0});\n'.format(var)
```

Comment:
Do we need normalize() here?

Reply:
Well no, but in this case n is useless and should be replaced with wposition.
`vec3 TextureCoordinate_Normal_res = n;`
I'm looking into that.

Reply:
The value of `TextureCoordinate_Normal_res` is probably generated in [`nodes_input.parse_texcoord()`](https://github.com/armory3d/armory/blob/34bf363e5f8cfeccde760eefe44d010a2b9c5a4c/blender/arm/material/cycles_nodes/nodes_input.py#L293-L294) (called from within [`cycles.parse_vector()`](https://github.com/armory3d/armory/blob/34bf363e5f8cfeccde760eefe44d010a2b9c5a4c/blender/arm/material/cycles.py#L305)), and the name is generated [here](https://github.com/armory3d/armory/blob/34bf363e5f8cfeccde760eefe44d010a2b9c5a4c/blender/arm/material/cycles.py#L534).